mailhog-command.php

```diff
@@ -10,3 +10,12 @@
 }
 
 EE::add_command( 'mailhog', 'Mailhog_Command' );
+
+EE::add_hook('before_invoke:mailhog up', function ( $args, $assoc_args ) {
```

Comment:
Do not use anonymous functions. I've already flagged this earlier and had a discussion why do not use.
ref: https://github.com/EasyEngine/easyengine/pull/1134#discussion_r206105926

Reply:
Added function callback